rails/rubocop.yml

```diff
@@ -9,3 +9,7 @@ Rails/UnknownEnv:
     - production
     - staging
     - demo
+
+Rails/SkipsModelValidations:
```

Comment:
i've been keeping this in alphabetical oder

Reply:
👍 can we lint our linter config!?!

Reply:
I can't find one but would be good!

Reply:
LOL lintception
if we really wanted to, we can add a quick ruby script (or w/e language) that loads in all the `rubcop.yml` files via `YAML.load_file` and runs through all the keys...